project/app/src/main/java/com/example/beacon/AuthManager.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.beacon
+
+object AuthManager {
+    private var authToken: String? = null
```

Comment:
My one concern is that we're gonna run into null tokens for some reason and that's gonna screw things up. I know it shouldn't happen if a user signs in correctly, but maybe change to the empty string just in case?

Reply:
good point, made the changes